resources/jscomposition/cases/casesDetail/edit.js

```diff
@@ -1,8 +1,366 @@
 import Vue from "vue";
 import CaseDetail from "./components/CaseDetail.vue";
+import Timeline from "../../../js/components/Timeline.vue";
+
+Vue.component("Timeline", Timeline);
 
 new Vue({
   el: "#case-detail",
   components: { CaseDetail },
-  data() {},
+  data() {
+    return {
+      activeTab: "pending",
+      showCancelRequest: false,
+      fieldsToUpdate: [],
+      jsonData: "",
+      selectedData: "",
+      monacoLargeOptions: {
+        automaticLayout: true,
+      },
+      showJSONEditor: false,
+      data: data,
+      requestId: requestId,
+      request: request,
+      files: files,
+      refreshTasks: 0,
+      canCancel: canCancel,
+      canViewPrint: canViewPrint,
+      status: "ACTIVE",
+      userRequested: [],
+      errorLogs: errorLogs,
+      disabled: false,
+      retryDisabled: false,
+      packages: [],
+      processId: processId,
+      canViewComments: canViewComments,
+      isObjectLoading: false,
+      showTree: false,
+      showTabs: true,
+      showInfo: true,
+    };
+  },
+  computed: {
+    activeErrors() {
```

Comment:
Where do we use this computed variables in the template?
 Is necesary this variables in the new implementation?